fury/ui.py

```diff
@@ -1792,6 +1792,7 @@ def __init__(self, center=(0, 0),
                  length=200, line_width=5,
                  inner_radius=0, outer_radius=10, handle_side=20,
                  font_size=16,
+                 axis=0,
```

Comment:
Can you replace this variable name, please? use `orientation="horizontal"` instead of `axis=0`.  It is more explicit and not all user are familiar with Numpy convention